zerver/lib/bulk_create.py

```diff
@@ -44,15 +44,15 @@ def bulk_create_users(realm: Realm,
             user_profile.email = get_display_email_address(user_profile)
         UserProfile.objects.bulk_update(profiles_to_create, ['email'])
 
-    user_ids = {user.id for user in profiles_to_create}
+    user_ids_and_active_status = {(user.id, user.is_active) for user in profiles_to_create}
```

Comment:
I would nitpick over the use of tuples.  I think we can just keep the `user_ids` local and then have the later loop just loop over `profiles_to_create`.